server/views/pages/induction/workedBefore/index.njk

```diff
@@ -10,7 +10,7 @@
     * backLinkUrl - url of the back link
     * backLinkAriaText - the aria label for the back link
     * form - form object containing the following fields:
-      * hasWorkedBefore - value for the "has worked before" question
+    * hasWorkedBefore - value for the "has worked before" question
```

Comment:
Very (very) minor, but the previous indentation here was meant to show that `hasWorkedBefore` is a property on the `form` object.
With the change in indentation here it could now be read that the `form` object contains both a `hasWorkedBefore` property AND an `errors` property